### ISSUE TITLE
pi_get_shows output format

### ISSUE THREAD

**Issue by MayaGans** [MayaGans/phishr#4]
Can we clean up the output format so that it doesn't include a list in the set_list column? Maybe use some `purrr` magic to turn it into a dataframe? Maybe with columns:

| Date | Venue | Title | Position | Set | Duration | Tags | 
|-----|-------|------|--------|-----|------|-----|

Would this exclude any of the data included in the call? 

I'd be happy to clean this up just want to discuss/get your thoughts @levisc8 before moving forward

**Comment by levisc8** [comment]
Yea, I think that'd be fine! Just to make sure I understand, you're thinking one row per song X show combination, rather than the single line per show?

**Comment by MayaGans** [comment]
Yeah, and then we can also include ID per song, that way the user can use those IDs easily for

```
pi_get_tracks(phishin_key, ID)
```

Thoughts?

**Comment by levisc8** [comment]
Makes sense. I probably won’t be able to get to that until later this week though. If we come up with a list of formatting changes, I don’t think it’ll take more than a night or two to make them (famous last words...).
